workers/social/lib/social/models/computeproviders/stacktemplate.coffee

```diff
@@ -368,12 +368,15 @@ module.exports = class JStackTemplate extends Module
       { group } = client.r
       query = { $set: { accessLevel } }
       id = @getId()
+      currentAccessLevel = @getAt 'accessLevel'
 
       @update query, (err) =>
 
         return callback err  if err
         return  unless group.slug
```

Comment:
it seems this line was also wrong, this should be `return callback null, this  unless group.slug`